frontend/src/page/admin/GuildDetails.tsx

```diff
@@ -0,0 +1,126 @@
+import {fetchGuildDetails, fetchGuildTags, GuildDetails as GuildDetailsDto} from "../../service/api";
+import {Base} from "../../component/Base";
+import {
+    Accordion,
+    AccordionDetails,
+    AccordionSummary,
+    Container,
+    Paper,
+    Stack,
+    TextField,
+    Typography
+} from "@mui/material";
+import {useParams} from "react-router-dom";
+import React, {Suspense, use, useEffect, useState} from "react";
+import {getGuildNameElement} from "../../guildUtil";
+import {DataGrid, GridColDef} from "@mui/x-data-grid";
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {parseISO} from "date-fns";
+import {format} from "date-fns/format";
+import {useDebounce} from "use-debounce";
+
+interface GuildDetailsDataProps {
+    dataPromise: Promise<GuildDetailsDto>
+}
+
+function FeaturesPaper({dataPromise}: GuildDetailsDataProps) {
+    const data = use(dataPromise).features;
+
+    const enabledFeatures = data.enabledFeatures.map(f => {
+        const enabledAt = format(parseISO(f.enabledAt), 'MM/dd/yyyy');
```

Comment:
```suggestion
    const dateFormat = new Intl.DateTimeFormat('en-GB', { dateStyle: 'short', timeStyle: 'short' })
    const enabledFeatures = data.enabledFeatures.map(f => {
        const enabledAt = dateFormat.format(new Date(f.enabledAt))
```
Also, using MM/DD/YYY is wrong.